src/main/content/_assets/js/guide-common.js

```diff
@@ -15,6 +15,12 @@ var threeColumnBreakpoint = 1440;
 var dep = false;
 var dep_closed = false;
 
+const preTags = document.querySelectorAll('pre');
+preTags.forEach(preTag => {
```

Comment:
Same as my comment on the other PR, please change this to a standard for loop

Reply:
Also, is it possible for you to rewrite this using the jQuery library (like the attr function you used later in the file)?  If you need any help or guidance, please reach out!